file-formats/swcr/type/zif_aff_swcr_v1.intf.abap

```diff
@@ -10,57 +10,52 @@ INTERFACE zif_aff_swcr_v1
       abap_language_version TYPE zif_aff_types_v1=>ty_abap_language_version,
     END OF ty_header.
 
-  "! <p class="shorttext">Relation Type</p>
-  "! Relation Type
-  "! $values {@link zif_aff_swcr_v1.data:co_relation_type}
-  TYPES ty_relation_type TYPE c LENGTH 1.
+  TYPES:
+    "! <p class="shorttext">Access Permission</p>
+    "! Access Permission
+    BEGIN OF ty_permission,
+      "! <p class="shorttext">Software Component</p>
+      "! Software Component
+      "! $required
+      software_component TYPE c LENGTH 30,
```

Comment:
Just a question: The permission type. Previously relation type isn't needed any more?

Reply:
That is correct. We now implicitly derive the permission type from the software component.